ci/kokoro/docker/build.sh

```diff
@@ -237,6 +244,9 @@ docker_flags=(
     # Additional flags to enable CMake features.
     "--env" "CMAKE_FLAGS=${CMAKE_FLAGS:-}"
 
+    # Tells scripts whether they are running as part of a CI or not.
+    "--env" "RUNNING_CI=${RUNNING_CI:-no}"
```

Comment:
Wouldn't the existing KOKORO environment variables suffice to say "we are running in CI" since we don't use any other type of CI?

Reply:
For today, it would. But my thinking went as follows:

1. The `KOKORO_JOB_NAME` var isn't passed to the docker env, so we'd still need to pass it (or something else)

2. `KOKORO_JOB_NAME` contains some specifically formatted string, so scripts would either need to know the format of that string.

3.`RUNNING_CI` is very specific, easy for scripts to parse, and would work if we ever had a CI system other than kokoro.

*shrug*

Convincing? :-)

Reply:
Yup, SGTM. I misread the PR by reading too quickly.